msticpy/data/drivers/mdatp_driver.py

```diff
@@ -40,6 +42,7 @@ def __init__(self, connection_str: str = None, instance: str = "Default", **kwar
 
         """
         super().__init__(**kwargs)
+        self._support_async = True
```

Comment:
Should we use  `DriverProp.SUPPORTS_ASYNC` instead to be consistent with the threaded approach?

Reply:
Yep - absolutely. I think I missed this